grpc/server/src/test/java/io/helidon/grpc/server/SslIT.java

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.grpc.server;
+
+import java.io.File;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import javax.net.ssl.SSLException;
+
+import io.helidon.config.Config;
+import io.helidon.config.ConfigSources;
+import io.helidon.grpc.server.test.Echo;
+import io.helidon.grpc.server.test.EchoServiceGrpc;
+
+import com.oracle.bedrock.runtime.LocalPlatform;
+import com.oracle.bedrock.runtime.network.AvailablePortIterator;
+import io.grpc.Channel;
+import io.grpc.StatusRuntimeException;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NegotiationType;
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import org.junit.AfterClass;
+import org.junit.Rule;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.rules.ExpectedException;
+import services.EchoService;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Tests for gRPC server with SSL connections
+ *
+ * @author Bin Chen
+ */
+public class SslIT {
+
+    // ----- data members ---------------------------------------------------
+
+    /**
+     * The {@link java.util.logging.Logger} to use for logging.
+     */
+    private static final Logger LOGGER = Logger.getLogger(SslIT.class.getName());
+
+    /**
+     * The Helidon {@link GrpcServer} being tested.
+     */
+    private static GrpcServer grpcServer_1WaySSL;
+
+    /**
+     * The Helidon {@link GrpcServer} being tested.
+     */
+    private static GrpcServer grpcServer_2WaySSL;
+
+    /**
+     * The Helidon {@link GrpcServer} being tested.
+     */
+    private static GrpcServer grpcServer_2WaySSLConfig;
+
+    /**
+     * Port used for 1waySSL
+     */
+    private static int m_nPort1WaySSL;
```

Comment:
1. I don't think we want to use `m_` prefix and Hungarian notation inside Helidon code, so I would just name the field `port1WaySSL`
2. Even if we did, it shouldn't be `m_` because the field is static, but we don't, so that's a moot point ;-)